libs/components/layout/src/lib/modules/box/box.component.html

```diff
@@ -9,7 +9,9 @@
   }"
 >
   <div class="sky-box-header-content">
-    <ng-content select="sky-box-header" />
+    <span #titleRef>
```

Comment:
Can this be an `ng-container` so we're not adding new elements to the DOM? I could see a consumer having a unit test that doesn't anticipate the child element.